blogs/posts/2024-02-11_sankey_plot.qmd

```diff
@@ -0,0 +1,565 @@
+---
+title: "Visualising participant recruitment with Sankey plots"
+author:
+  - Craig Parylo
+date: "2024-02-11"
+categories: [learning, tutorial, visualisation]
+reference-location: margin
+citation-location: margin
+---
+
+```{r}
+#| code-fold: true
+#| warning: false
+#| message: false
+#| output: false
+
+# libraries
+library(tidyverse)       # 'tidy' data wrangling
+library(plotly)          # sankey visualisation framework
+library(reactable)       # viewing interactive datatables
+library(glue)            # concatenating strings
+library(tidygraph)       # api for graph / network manipulation
+library(scales)          # used for colour transformation
+```
+
+# Introduction
+
+Sankey diagrams are great tools to visualise flows through a system. They show the flow / conversion rate after each step of a process and the width of the arrows is proportional to the flow / conversion rate.
+
+I'm working on a project to evaluate the effectiveness of a risk screening programme for people aged between 55-74 years and a history of smoking[^1]. One report requirement is to be able to visualise how people have engaged with the programme, from recruitment, attendance at appointments, their outcome from risk assessment, low-dose CT scans and will eventually cover the impact of the screening on early detection of those diagnosed with lung cancer.
```

Comment:
Probably best to expand acronyms/initialisms throughout the article when first used. I'd argue this might not be necessary for 'CT', since these are generally known as 'CT scans' by the general public! Might be more useful in cases like 'TLHC' in the first footnote/first mention in the article body.